### PR TITLE
IGVF-3-simplify-user-json

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/schemas/user.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
             "title": "Lab",
             "description": "Lab user is primarily associated with.",
             "comment": "See lab.json for available identifiers.",
-            "permission": "import-items",
```
Comment (cricketsloan) [inline src/igvfd/schemas/user.json:38]: @gabdank I don't remember catching this.  Was the permissions working on ENCODEd at all with this - versus _

File: src/igvfd/schemas/user.json [hunk 2]
```diff
@@ -86,12 +86,30 @@
...
+                "enum": [
+                    "Primary Investigator",
+                    "Project Manager",
+                    "Submitter",
+                    "Post Doc",
+                    "Data Wrangler",
+                    "Scientist",
+                    "Computational Scientist",
+                    "Software Developer",
+                    "Other"
```
Comment (cricketsloan) [inline src/igvfd/schemas/user.json:102]: @khzlin do we have a lot of graduate students or students as users?  I am also wondering what we call the NHGRI folks.  What do we call them now?

Comment (khzlin) [reply src/igvfd/schemas/user.json:102]: I think "trainees" are used quite a lot to describe students a lot in calls. As for NHGRI folks, I'm not sure how to call them. Maybe "NHGRI Scientific Program Analyst"?

File: src/igvfd/schemas/user.json (hunk 1 above)
Comment (gabdank) [reply src/igvfd/schemas/user.json:38]: @cricketsloan we discovered that ENCODE allows "permission" to be ANY string. I just suggest to be consistent with other places using import items (import_items)

File: src/igvfd/schemas/user.json (hunk 2 above)
Comment (gabdank) [reply src/igvfd/schemas/user.json:102]: I would like to NOT use Trainees - due to a discussion of how to call the "young generation of researchers" that is happening at slack for IGVF

**Review by gabdank — COMMENTED** [review]
@cricketsloan In general, I don't think we need upgrade scripts for these.. we are just creating the first version of Lab/User/Award

File: src/igvfd/schemas/user.json (hunk 2 above)
Comment (cricketsloan) [reply src/igvfd/schemas/user.json:102]: What about NHGRI Staff Member and is Student offensive?  I think we can add Post Doc.

**Comment by cricketsloan** [comment]
Looks like adding a few more terms for people, updating the changelog to state where this broke from ENCODEd(This is version 1 of IGVF and it varies from version X of encodeD in the following ways.)

File: src/igvfd/schemas/user.json (hunk 2 above)
Comment (khzlin) [reply src/igvfd/schemas/user.json:102]: I like "NHGRI Staff Member". Is it OK to proceed without a term for Student for now? @gabdank @cricketsloan

Comment (cricketsloan) [reply src/igvfd/schemas/user.json:103]: yes we can leave out student and get this done

**Review by cricketsloan — APPROVED** [review]

**Review by cricketsloan — APPROVED** [review]
Sorry, I had tried to review this from my phone previously, but apparently that did not save

**Comment by keenangraham** [comment]
@khzlin Can you rebase/resolve conflicts when you get chance?

**Comment by khzlin** [comment]
@keenangraham OK, rebased and tests passed.

**Pull request merged**